Auxiliary/DirectXTexHelper.cpp

```diff
@@ -0,0 +1,58 @@
+//--------------------------------------------------------------------------------------
+// File: DirectXTexHelper.cpp
+//
+// DirectXTex helper functions for JPEG/PNG sources
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//--------------------------------------------------------------------------------------
+#include "DirectXTexP.h"
+
+#include <cstdio>
```

Comment:
Sort these alphabetically plz.

Reply:
Got it. I think it will be helpful if there is a .clang-format. I mean, in the future.